meorg_client/constants.py

```diff
@@ -0,0 +1,27 @@
+"""Constants."""
+
+# Valid HTTP methods
+VALID_METHODS = ["POST", "GET", "DELETE", "PUT"]
```

Comment:
Declaring const outside of array can be more useful as they can be used individually at multiple places.

Reply:
Done, coming in forthcoming commit.